packer-plugin-sdk/packer/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Package packer contains all of the interfaces for key Packer objects.
+
+This module will need to be imported by all but the very simplest plugins. It
+represents the foundation of the API that the Core and Plugins use to
+communicate with each other.
+
+Changes to any of the interfaces in this package likely represent a
+backwards-incompatibility and should therefore only be made rarely and when
+absolutely necessary.
+
+Plugins will need to implement either the Builder, Provisioner,
+or Post-Processor interfaces, and will likely create an Artifact. The
+Communicator must be implemented in the Builder and is then passed into the
```

Comment:
```suggestion
Communicator must be implemented in the Builder and then passed into the
```
Please, ignore if this change doesn't make sense.